force-app/main/default/lwc/collapsableSection/collapsableSection.html

```diff
@@ -0,0 +1,20 @@
+<template>
+    <div class="slds-section" data-id={id}>
+        <!--section header-->
+        <h3 >
+            <button type="button" class="section-header" aria-controls={id}  data-buttonid={id}
```

Comment:
Bør ha en aria-expanded her

Reply:
Jeg setter dette i event
 if(currentsection.className.search('slds-is-open')==-1){
            console.log(currentsection);
            currentsection.className = 'slds-section slds-is-open';
        }else{
            currentsection.className = 'slds-section slds-is-close';
        }


Har prøvd forskjellige måter å sette aria-expanded, men får ikke trigget den - dvs. den vil ikke åpne seg. 
Et eks.
let currentsection = this.template.querySelector('[data-buttonid="'+buttonid + '"]');
        console.log(currentsection);
       if(currentsection.ariaExpanded ==="true"){
            console.log("går inn her"); // Den går inn her. Noe i strukturen her jeg ikke har klart for meg.
            currentsection.setAttribute('aria-expanded', 'true')
        }else{
            currentsection.setAttribute('aria-expanded', 'false')
        }